build/makelib/golang.mk

```diff
@@ -156,6 +156,14 @@ go.test: go.vet go.fmt $(CGO_PREREQS)
 #	@$(GOHOST) test -v -i $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 	@$(GOHOST) test -cover $(GO_PKG_FLAGS) $(GO_NONSTATIC_FLAGS) $(GO_ALL_PACKAGES)
 
+.PHONY: go.lint
+go.lint:
```

Comment:
Yes you are right but i want to start slow. Im sure we will see a lot of errors and warnings. I want to introduce lint checks first and once we have resolve all that, promote it to voting and then look to add vet checks